README.md

```diff
@@ -97,6 +97,25 @@ npm run build
 
 Generated files of the website go to the `.next` folder.
 
+### Running the Docker image
+
+To run the Docker image, simply run the following command:
+
+### Steps to start server
+
+- `cd` into the repo
+- run the following command `docker build -t async_image -f Dockerfile . && docker run -d -p 3000:3000 async_image`
+- Visit localhost:3000 and the website should be live
+
+### For Windows 
+- `cd` into the repo 
+- run the command one by one `docker build -t async_image -f Dockerfile.dev .` then `docker run -d -p 3000:3000 async_image`
```

Comment:
I guess we do not need windows specific instructions, right? also `dockerfile.dev` no longer here